src/Raven.Server/Https/AuthenticatingAdapter.cs

```diff
@@ -108,7 +110,9 @@ public async Task<IAdaptedConnection> OnConnectionAsync(ConnectionAdapterContext
 
             var tls = context.Features.Get<ITlsConnectionFeature>();
             var certificate = tls?.ClientCertificate;
-            var authenticationStatus = _server.AuthenticateConnectionCertificate(certificate);
+            var conn = context.Features.Get<IHttpConnectionFeature>();
+            var address = $"{conn?.RemoteIpAddress}:{conn?.RemotePort}";
```

Comment:
Why do we need to get the address here? I assume that this is for logging / error generation?
If so, better to pass these details to the `AuthenticateConnectionCertificate` method.
I want to avoid this allocation if possible (most connections are successful)

Reply:
OK